tools/setup_kube.sh

```diff
@@ -50,12 +50,16 @@ fi
 # Bookinfo requires more memory
 minikube config set memory 4096
 
+# Get skaffold to deploy online boutique on minikube
+# TODO: is glcloud automatically installed above?  it should be
+gcloud components install skaffold
```

Comment:
Is this still needed? I think it is better to work with raw yamls in the scripts. They are easier to debug.